ufs_cmds.c

```diff
@@ -1099,7 +1099,9 @@ int do_attributes(struct tool_options *opt)
 			rc = do_query_rq(fd, &bsg_req, &bsg_rsp,
 					UPIU_QUERY_FUNC_STANDARD_READ_REQUEST,
 					UPIU_QUERY_OPCODE_READ_ATTR, att_idn,
-					opt->index, opt->selector, 0, 0, 0);
+					tmp->device_level & ARRAY ?
+					opt->index : 0,
+					opt->selector, 0, 0, 0);
```

Comment:
I think this is redundant fix, By default in case and user didn't set index, selector the values are 0. 
See  verify_arg_and_set_default function